fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -964,6 +968,22 @@ cleanTabletSchedQClause
     : CLEAN TABLET SCHEDULER QUEUE
     ;
 
+decommissionDiskClause
```

Comment:
Should make the disk decommission and disk disable cannot run simutaneourly.

Reply:
done